core/src/main/java/io/neow3j/protocol/core/methods/response/StackItem.java

```diff
@@ -0,0 +1,187 @@
+package io.neow3j.protocol.core.methods.response;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import io.neow3j.model.types.StackItemType;
+import io.neow3j.protocol.core.methods.response.StackItem.StackDeserializer;
+import io.neow3j.utils.Numeric;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+@JsonDeserialize(using = StackDeserializer.class)
+public class StackItem {
+
+    protected StackItemType type;
+    protected Object value;
+
+    public StackItem(StackItemType type, Object value) {
+        this.type = type;
+        this.value = value;
+    }
+
+    public StackItemType getType() {
```

Comment:
JavaDocs?! Makes sense to add it here?

I'm not sure if JavaDocs from an extended class (with @Override annotation) will inherit the JavaDocs as well, or not? Check that.